test/bind.spec.js

```diff
@@ -376,36 +376,7 @@ test('classes are removed before being added', async () => {
     })
 });
 
-test('cursor position is preserved on selectable text input', async () => {
```

Comment:
So does this behaviour still work? Just the unit test fails?

Reply:
I believe that test had a bug. (also the implementation didn't make sense to me). I tried my way of test and it did past but since there was no any way of setting cursor position by code, i removed that test. Open to another way of testing cursor position or discussing the previous way to me get it clear.

Reply:
which test fails? 
```js
cursor position is preserved on selectable text input

    expect(received).toEqual(expected) // deep equality

    Expected: 3
    Received: 0

      398 |         expect(document.querySelector('input').value).toEqual('baz')
      399 |         expect(document.querySelector('input').selectionStart).toEqual(0)
    > 400 |         expect(document.querySelector('input').selectionEnd).toEqual(3)
          |                                                              ^
      401 |         expect(document.querySelector('input').selectionDirection).toEqual('backward')
      402 |     })
      403 | })
```
If you look at the test, it's really confusing. I don't know, maybe i am wrong.

Reply:
The test is not immediate but it does what it says. 
It has a listener on the select function, it simulates the user selection, the listener updates the model value and the model triggers the input update changing from bar to baz. At the end the test checks that, despite the value being changed. the selection is still the same.

It's not a real case imo because the model value would only change if you take some actions (like clicking buttons or similar) so the input would lose the focus. Also, since the content is changed, your old selection is likely to be wrong anyway.

This implementation seems neater but since there have been a lot of back and forward on this topic, can we make sure that it works correctly for each input fields in all the major browsers please (chrome, firefox, safari and edge)?
Manual tests are okay since we can't really have real automated tests for browsers. Thanks

Reply:
see https://codepen.io/SimoTod/pen/LYpyjzz, if you select the test it changes but the selection disappear. To me, it's acceptable but the guys who wrote the test thought it was a bug so I guess it's subjected to personal opinions

Reply:
> see https://codepen.io/SimoTod/pen/LYpyjzz, if you select the test it changes but the selection disappear. To me, it's acceptable but the guys who wrote the test thought it was a bug so I guess it's subjected to personal opinions

I just tried your sample with Vue js, it behaves the same.